src/main/docker/Dockerfile.jvm

```diff
@@ -21,34 +21,36 @@
 # docker run -i --rm -p 8080:8080 -p 5005:5005 -e JAVA_ENABLE_DEBUG="true" quarkus/release-manager-jvm
 #
 ###
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
 
 ARG JAVA_PACKAGE=java-11-openjdk-headless
 ARG RUN_JAVA_VERSION=1.3.8
 
 ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
 
 # Install java and the run-java script
-# Also set up permissions for user `1000`
+# Also set up permissions for user `1001`
```

Comment:
Why did you change the userid?

Reply:
User UID should be `1000` as needed by jenkins jnlp agent - see here for more information: https://github.com/jenkinsci/kubernetes-plugin#pipeline-sh-step-hangs-when-multiple-containers-are-used

Reply:
I didn't actually change the user, this `Dockerfile.jvm` is the old `fast-jar` file that has became the default Dockerfile since 1.12 Quarkus version, so Dockerfile.jvm. It contained already the user `1001` and as I didnt know that the `1000` user was needed, I kept the `1001`. I will put the `1001` everywhere.

Reply:
+1